### DISCUSSION TITLE
My last year's HashCode Experience

### DISCUSSION THREAD

**Discussion by jai-dewani** [EddieHubCommunity/support#824] [General]
I had a lot of fun participating in hashcode last year, I and 3 of my friends got together to participate in this one of a kind event which lasts 4 hours. 
>This was my first time participating in google hashcode. Since I didn’t even know this competition existed till January 2020. After going through the official website I thought it’s is just another Algorithmic Team-based Competitive programming contest but turns out hashcode is one of its kind of contest where you get to solve real-life problems faced by engineers at google whose perfect solutions can't be found even with today's computation power so you have to aim for the best possible solution and the more close you get to that perfect solution, the more points you earn during the contest.

Check out my whole experience [here](https://blogs.jaid.tech/hashcode-2020/). 

How was your last year's experience, and are you going to participate this year? We can create teams within the community if people are still looking out for teammates!

Read more about hashcode [here](https://codingcompetitions.withgoogle.com/hashcode/about)

**Comment by eddiejaoude** [comment]
Thanks for sharing @jai-dewani 👍 .

Here are some snippets from the shared links...

>This was my first time participating in google hashcode. Since I didn’t even know this competition existed till January 2020. After going through the official website I thought it’s is just another Algorithmic Team-based Competitive programming contest but turns out hashcode is one of its type of contest where you get to solve a real-life problem which is faced by engineers at google which at this moment can’t be solved exactly so you have to aim for the best possible solution and the more close you get to that perfect solution, the more points you earn during the contest.


> Google’s team-based programming competition, Hash Code, allows you to share your skills and connect with other coders as you work together to solve a problem modeled off a real Google engineering challenge! In small teams of two to four, coders all over the world will tackle the first problem through Online Qualifications. Though this round is hosted online, teams can come together virtually to compete side-by-side in locally coordinated Hash Code Hubs. The top teams from this round are invited to join us for the virtual World Finals.
